src/controllers/main-controller.ts

```diff
@@ -169,6 +170,68 @@ router.get(
   }),
 );
 
+router.get(
+  '/tickers',
+  asyncRoute(async (req: Request, res: Response) => {
+    const chainId = validateChainId(req, true);
+
+    const ammPools = await db
```

Comment:
this table has only rsk network data, for bob, we need to take pool data from controllers/sdex/index.ts - it's on chain data